src/ClientData/CaptureDataTest.cpp

```diff
@@ -46,6 +50,44 @@ constexpr std::array<uint64_t, kTimersForSecondId> kSortedDurationsForSecondId =
 
 constexpr uint64_t kLargeInteger = 10'000'000'000'000'000;
 
+constexpr uint64_t kFirstTid = 1000;
+constexpr uint64_t kSecondTid = 2000;
+constexpr uint64_t kNonExistingTid = 3000;
+constexpr uint64_t kStTimestamp1 = 50;
+constexpr uint64_t kEnTimestamp1 = 100;
+constexpr uint64_t kStTimestamp2 = 100;
+constexpr uint64_t kEnTimestamp2 = 150;
+constexpr uint64_t kStTimestamp3 = 150;
+constexpr uint64_t kEnTimestamp3 = 200;
+constexpr uint32_t kWakeupTid = 4200;
+constexpr uint32_t kWakeupPid = 420;
+constexpr uint32_t kInvalidPidAndTid = 0;
+
+const ThreadStateSliceInfo kSlice1{
+    kFirstTid,        orbit_grpc_protos::ThreadStateSlice::kInterruptibleSleep, kStTimestamp1,
+    kEnTimestamp1,    ThreadStateSliceInfo::WakeupReason::kNotApplicable,       kInvalidPidAndTid,
+    kInvalidPidAndTid};
+const ThreadStateSliceInfo kSlice2{
+    kFirstTid,     orbit_grpc_protos::ThreadStateSlice::kRunnable, kStTimestamp2,
+    kEnTimestamp2, ThreadStateSliceInfo::WakeupReason::kUnblocked, kWakeupTid,
+    kWakeupPid};
+const ThreadStateSliceInfo kSlice3{
+    kFirstTid,        orbit_grpc_protos::ThreadStateSlice::kRunning,      kStTimestamp3,
+    kEnTimestamp3,    ThreadStateSliceInfo::WakeupReason::kNotApplicable, kInvalidPidAndTid,
+    kInvalidPidAndTid};
+const ThreadStateSliceInfo kSlice4{
+    kSecondTid,       orbit_grpc_protos::ThreadStateSlice::kInterruptibleSleep, kStTimestamp1,
+    kEnTimestamp1,    ThreadStateSliceInfo::WakeupReason::kNotApplicable,       kInvalidPidAndTid,
+    kInvalidPidAndTid};
+const ThreadStateSliceInfo kSlice5{
+    kSecondTid,    orbit_grpc_protos::ThreadStateSlice::kRunnable, kStTimestamp2,
+    kEnTimestamp2, ThreadStateSliceInfo::WakeupReason::kUnblocked, kWakeupTid,
+    kWakeupPid};
+const ThreadStateSliceInfo kSlice6{
```

Comment:
Slices are really similar and we are testing the same functionality several times. I'd rather have only one slice with `kSecondTid` just to test that getting a different thread_id works.